pages/checkout-form.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Tinos:wght@400;700&display=swap" rel="stylesheet">
+    <title>Checkout</title>
+</head>
+<body class="checkout_form">
+<div class="wrapper">
+    <div class="payment_wrapper">
+        <div class="payment_inner" id="payment_wrapper">
+            <div class="payment_logo">
+                <span class="logo"></span>
+                <h2 class="book_title">Checkout form</h2>
+            </div>
+
+            <div class="cart_container"></div>
+            <div class="cart_total"></div>
+
+            <form   class="delivery" id="delivery" name="delivery">
+                <div class="text_form">
+                    <div class="input_element">
+                        <label for="input-name">Name</label>
```

Comment:
Another rule of thumb is to let the user know which inputs are required to fill, usually adding `*` at the end of label
```suggestion
                        <label for="input-name">Name*</label>
```